### DISCUSSION TITLE
Allow relative urls for Api urls (e.g. Json Data Provider)

### DISCUSSION THREAD

**Discussion by robertjf** [leekelleher/umbraco-contentment#212] [Idea]
Currently to retrieve data from a custom Api controller we need to provide the full url - for example:

```
https://localhost:44340/umbraco/backoffice/lsc/crm/therapists
```

It would be useful to be able to provide a domain agnostic url mapped to the current domain, e.g.:

```
/umbraco/backoffice/lsc/crm/therapists
```

Which would be mapped to the domain that Umbraco is running from.

In addition, currently if the ApiController inherits from UmbracoAuthorisedApiController, the data cannot be retrieved.  I had to open the controller endpoint above up and remove the authorization in order for Contentment to retrieve it.

**Comment by leekelleher** [comment]
I think this is the same question as #92. 😊

My reply on there... https://github.com/leekelleher/umbraco-contentment/discussions/92#discussioncomment-629299

**Comment by robertjf** [comment]
Thanks @leekelleher - that will do :)
